packages/lesmis-server/src/connections/ReportConnection.ts

```diff
@@ -27,4 +28,8 @@ export class ReportConnection extends SessionHandlingApiConnection {
       period: periods.join(';'),
     });
   }
+
+  async forwardReportQuery(reportCode: string, query: QueryParameters) {
```

Comment:
Is anything else using the more opinionated one above? I'd say we shouldn't have both. Either replace the `fetchReport` completely (and probably keep that name) or make the report route use the opinionated one

Reply:
The more opinionated `fetchReport` is not being used by `lesmis-server`, looks like this file was originally copy/pasted from PSSS where that was actually used. I've replaced it with the new less opinionated function.